src/Ardalis.Specification/SpecificationEvaluator.cs

```diff
@@ -1,12 +1,11 @@
-﻿using Microsoft.EntityFrameworkCore;
-using System.Linq;
+﻿using System.Linq;
 using System.Threading.Tasks;
 
 namespace Ardalis.Specification
 {
     public class SpecificationEvaluator<T> where T : class
     {
-        public static async Task<IQueryable<T>> GetQuery(IQueryable<T> inputQuery, ISpecification<T> specification)
+        public static Task<IQueryable<T>> GetQuery(IQueryable<T> inputQuery, ISpecification<T> specification)
```

Comment:
Why did you remove async here?

Reply:
@ardalis 
Good question, the method does not have any asynchronous operation like network calls and hence not required to add a 'async', However, if you want to consume on the other side we can still call the method like 'await GetQuery()' as it support Task<T>. Visual studio was showing warning that's why I have to make this change..